src/lib/i18n/dictionaries/englishDictionary.ts

```diff
@@ -1696,6 +1699,15 @@ const decorationDescriptions: Record<DecorationDescriptions, string> = {
     "A display of allegiance to the Nightshade Faction",
   "description.bumpkin.faction.banner":
     "A display of allegiance to the Bumpkin Faction",
+
+  // Clash of Factions
+  "description.turbo.sprout":
+    "An engine that boosts the Green House's growth speed by 50%.",
```

Comment:
```suggestion
  "description.turbo.sprout":
    "An engine that reduces the Green House's growth time by 50%.",
```
Standardise with other time reduction boosts